### ISSUE TITLE
--engine throws Engine has not resolved

### ISSUE THREAD

**Issue by MiguelDomingues** [marp-team/marp-cli#240]
With version 0.18.0 I was able to use the following custom engine:

```javascript
const { Marp } = require('@marp-team/marp-core')
const markdownItAttrs = require('markdown-it-attrs')
module.exports = opts => new Marp(opts).use(markdownItAttrs)
```

by using the following commands:
```
npm install @marp-team/marp-core markdown-it-attrs --save-dev
marp --engine ./engine.js slide-deck.md
```

With versions 0.18.1 and 0.18.2 I get the following error: `[ ERROR ] The specified engine has not resolved.`

Adding the "import-from" back to the "dependencies" of package.json solves the issue.

**Comment by yhatt** [comment]
Thanks for your report. To reduce dependencies, we had bundled some modules in #233 (>= v0.18.1). But dynamic `require()` by `import-from` throws error by rollup: `Error: Dynamic requires are not currently supported by @rollup/plugin-commonjs`.

```
Error: Dynamic requires are not currently supported by @rollup/plugin-commonjs
    at require (/Users/yhatt/Programs/marp-team/marp-cli/lib/marp-cli.js:102:8)
    at Function.silent$2 [as silent] (/Users/yhatt/Programs/marp-team/marp-cli/node_modules/import-from/index.js:8:10)
    at /Users/yhatt/Programs/marp-team/marp-cli/src/engine.ts:37:46
    at Array.some (<anonymous>)
    at Function.resolveModule (/Users/yhatt/Programs/marp-team/marp-cli/src/engine.ts:33:17)
    at /Users/yhatt/Programs/marp-team/marp-cli/src/engine.ts:23:26
    at Generator.next (<anonymous>)
    at /Users/yhatt/Programs/marp-team/marp-cli/node_modules/tslib/tslib.es6.js:74:71
    at new Promise (<anonymous>)
    at __awaiter (/Users/yhatt/Programs/marp-team/marp-cli/node_modules/tslib/tslib.es6.js:70:12)
```

**Comment by yhatt** [comment]
Fixed in the latest release v0.18.3. Sorry for the inconvenience!
